packages/components/src/components/password-input/password-input.jsx

```diff
@@ -36,6 +38,7 @@ const PasswordInput = ({
 
 PasswordInput.propTypes = {
     className: PropTypes.string,
+    autoComplete: PropTypes.string.isRequired,
```

Comment:
Do we want to make it required?
The component will still work fine without the `prop`, it's just the warning messages.